engine/apps/alerts/paging.py

```diff
@@ -134,107 +87,40 @@ def _trigger_alert(
     return alert.group
 
 
-def check_user_availability(user: User) -> typing.List[AvailabilityWarning]:
-    """Check user availability to be paged.
-
-    Return a warnings list indicating `error` and any additional related `data`.
-    """
-    warnings: typing.List[AvailabilityWarning] = []
-    if not user.notification_policies.exists():
-        warnings.append(
-            {
-                "error": PagingError.USER_HAS_NO_NOTIFICATION_POLICY,
-                "data": {},
-            }
-        )
-
-    is_on_call = False
-    schedules = OnCallSchedule.objects.filter(
-        Q(cached_ical_file_primary__contains=user.username) | Q(cached_ical_file_primary__contains=user.email),
-        organization=user.organization,
-    )
-    schedules_data: ScheduleWarnings = {}
-    for s in schedules:
-        # keep track of schedules and on call users to suggest if needed
-        oncall_users = list_users_to_notify_from_ical(s)
-        schedules_data[s.name] = set(u.public_primary_key for u in oncall_users)
-        if user in oncall_users:
-            is_on_call = True
-            break
-
-    if not is_on_call:
-        # user is not on-call
-        # TODO: check working hours
-        warnings.append(
-            {
-                "error": PagingError.USER_IS_NOT_ON_CALL,
-                "data": {"schedules": schedules_data},
-            }
-        )
-
-    return warnings
```

Comment:
we no longer need this (nor the related API endpoint) since there is now an attribute on the `User` model, `is_currently_oncall`, that achieves this same functionality